src/hiveboard_bridge/test/MessageHandlerUnitTest.cpp

```diff
@@ -0,0 +1,120 @@
+#include "hiveboard_bridge/MessageHandler.h"
+
+#include <functional>
+#include <gmock/gmock.h>
+#include <hivemind-host/FunctionCallArgumentDTO.h>
+#include <hivemind-host/FunctionCallRequestDTO.h>
+#include <hivemind-host/MessageDTO.h>
+#include <hivemind-host/RequestDTO.h>
+#include <optional>
+
+bool g_testFunctionCalled = false;
+int g_testValue1 = 0;
+int g_testValue2 = 12;
```

Comment:
I would highly recommend putting them in the fixture, this way you are sure they are retested  between each tests, even if google changes their implementation. I would maybe avoid the need to reset them in the setup